localstack/utils/aws/arns.py

```diff
@@ -28,12 +28,16 @@ def sqs_queue_url_for_arn(queue_arn):
         arn = parse_arn(queue_arn)
         region_name = arn["region"]
         queue_name = arn["resource"]
+        account_id = arn["account"]
     except InvalidArnException:
         region_name = None
         queue_name = queue_arn
+        account_id = DEFAULT_AWS_ACCOUNT_ID
 
     sqs_client = connect_to_service("sqs", region_name=region_name)
-    result = sqs_client.get_queue_url(QueueName=queue_name)["QueueUrl"]
+    result = sqs_client.get_queue_url(QueueName=queue_name, QueueOwnerAWSAccountId=account_id)[
```

Comment:
nit: I could also use a `kwargs` dictionary and not add this parameter to the call, but I think the `sqs_client` will always be created with the `DEFAULT_AWS_ACCOUNT_ID`, right?

Reply:
Being explicit with account IDs is the way forward to achieve 100% multi-tenancy in LocalStack 🙌